### PR TITLE
PDF Export DataGrid - add doс comments for arguments of the 'exportDataGrid' function

### PR DESCRIPTION
**Pull request by EugeniyKiyashko**

### CODE REVIEW

File: js/exporter/jspdf/export.js [hunk 1]
```diff
@@ -110,7 +110,7 @@ export const Export = {
                 jsPDFDocument.autoTable.__autoTableOptions = autoTableOptions;
                 ///#ENDDEBUG
 
-                resolve(jsPDFDocument);
+                resolve(jsPDFDocument); // Do we need to return the Promise with jsPDFDocument param?
```
Comment (IgnatovDan) [inline js/exporter/jspdf/export.js]: +1
It's in input args and should not be passed as the only result of our primise:

```
    export: function(options) {
        const {
            jsPDFDocument,
```
Change to "resolve()"?

File: js/pdf_exporter.d.ts [hunk 2]
```diff
@@ -0,0 +1,67 @@
+import dxDataGrid from './ui/data_grid';
+
+/**
+* @docid PdfExportBaseProps
+* @namespace DevExpress.pdfExporter
+* @type object
+* @hidden
+*/
+export interface PdfExportBaseProps {
```
Comment (IgnatovDan) [inline js/pdf_exporter.d.ts]: imho we should not split interface into Base and DataGrid props.
It's easy to split it when we will add Pivot props

File: ts/dx.all.d.ts [hunk 3]
```diff
@@ -2975,6 +2975,42 @@ declare module DevExpress.fx {
      */
     export function stop(element: Element, jumpToEnd: boolean): void;
 }
+declare module DevExpress.pdfExporter {
+    /**
+     * [descr:PdfExportBaseProps]
+     */
+    export interface PdfExportBaseProps {
```
Comment (IgnatovDan) [inline ts/dx.all.d.ts]: Another excessive split to Base and Grid props.

**Review by IgnatovDan — CHANGES_REQUESTED** [review]

File: js/exporter/jspdf/export.js (hunk 1 above)
Comment (EugeniyKiyashko) [reply js/exporter/jspdf/export.js]: Yes. I also think that we must to call resolve() with empty value. Fixed in [b50f5c4](https://github.com/DevExpress/DevExtreme/pull/14787/commits/b50f5c4d3384b10050d24ca9d14d4e1295198079)

File: js/pdf_exporter.d.ts (hunk 2 above)
Comment (EugeniyKiyashko) [reply js/pdf_exporter.d.ts]: I fully agree with you. We don't want to get ahead of ourselves here. I merged two interfaces into one. https://github.com/DevExpress/DevExtreme/pull/14787/commits/27a4e5e7240e73d918f2db63bf62930dec56dfa3

File: ts/dx.all.d.ts (hunk 3 above)
Comment (EugeniyKiyashko) [reply ts/dx.all.d.ts]: It is autogenerated part of the merge after update-ts was called.

**Review by IgnatovDan — APPROVED** [review]

**Review by jsdmitry — APPROVED** [review]

**Pull request merged**